<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rootWidget</class>
 <widget class="QWidget" name="rootWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>40</y>
     <width>951</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Здраствуйте, Name Surname</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>350</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сделать заказ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>500</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ordersButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>250</y>
     <width>350</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Список заказов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>createButton</sender>
   <signal>clicked()</signal>
   <receiver>rootWidget</receiver>
   <slot>createOrder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ordersButton</sender>
   <signal>clicked()</signal>
   <receiver>rootWidget</receiver>
   <slot>switchToOrdersList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>rootWidget</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createOrder()</slot>
  <slot>switchToOrdersList()</slot>
  <slot>exit()</slot>
 </slots>
</ui>
